tools/packaging/ubi-rpm.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN echo $PATH
 
 # Get and install ArmoniK api into the image
 WORKDIR /tmp
-ARG API_VERSION=c8c4c5ce09dcbdc677ff5c630e49521e5141ae70
+ARG API_VERSION=3.13.0
```

Comment:
The fact that you have to change it this many time is not great

Reply:
Should be an one or two places max I agree